doc/cephadm/troubleshooting.rst

```diff
@@ -176,3 +175,28 @@ To access the admin socket, first enter the daemon container on the host::
 
     [root@mon1 ~]# cephadm enter --name <daemon-name>
     [ceph: root@mon1 /]# ceph --admin-daemon /var/run/ceph/ceph-<daemon-name>.asok config show
+
+
+Restoring the MON quorum
+------------------------
+
+In case the Ceph MONs cannot form a quorum, cephadm is not able
```

Comment:
A "ceph orch pause"  before and after execute the procedure could be healthy.

Reply:
if the MONs are not in quorum, there is nothing you can do about it. `ceph orch pause` will fail. `ceph config-key set mgr/cephadm/pause` will also fail.